tests/ipex/test_modeling.py

```diff
@@ -354,20 +373,27 @@ def test_compare_to_transformers(self, model_arch):
             transformers_outputs = transformers_model(**inputs)
         outputs = ipex_model(**inputs)
 
+        # Test re-load model
         with tempfile.TemporaryDirectory() as tmpdirname:
             ipex_model.save_pretrained(tmpdirname)
             loaded_model = self.IPEX_MODEL_CLASS.from_pretrained(tmpdirname)
             loaded_model_outputs = loaded_model(**inputs)
+
+        # Test init method
+        init_model = self.IPEX_MODEL_CLASS(transformers_model)
+        init_model_outputs = init_model(**inputs)
+
         # Compare tensor outputs
         self.assertTrue(torch.allclose(outputs.logits, transformers_outputs.logits, atol=1e-3))
         self.assertTrue(torch.equal(outputs.logits, loaded_model_outputs.logits))
+        self.assertTrue(torch.equal(outputs.logits, init_model_outputs.logits))
 
     @parameterized.expand(SUPPORTED_ARCHITECTURES)
     def test_pipeline(self, model_arch):
         model_id = MODEL_NAMES[model_arch]
         model = self.IPEX_MODEL_CLASS.from_pretrained(model_id, export=True)
         preprocessor = AutoFeatureExtractor.from_pretrained(model_id)
-        pipe = pipeline("audio-classification", model=model, feature_extractor=preprocessor)
+        pipe = pipeline("audio-classification", model=model, feature_extractor=preprocessor, framework="pt")
```

Comment:
why is this needed ?
```suggestion
        pipe = pipeline("audio-classification", model=model, feature_extractor=preprocessor)
```